crates/services/txpool_v2/src/pool_worker.rs

```diff
@@ -516,7 +525,7 @@ where
         }
     }
 
-    fn remove_and_coin_dependents(&mut self, tx_ids: (Vec<TxId>, Error)) {
+    fn expired_transactions_and_dependents(&mut self, tx_ids: (Vec<TxId>, Error)) {
```

Comment:
i think we can take an iterator here too, no?

Reply:
I don't like the type transformation it requires to write to make this works when calling this function because of the tuple, I would pass this one.